hail/python/hailtop/utils/utils.py

```diff
@@ -107,7 +107,7 @@ def digits_needed(i: int) -> int:
     return 1 + digits_needed(i // 10)
 
 
-def grouped(n, ls):
+def grouped(n: int, ls: List[Any]) -> Iterable:
```

Comment:
Let's go with `Iterable[Any]`. `Iterable` is just like `List`: it's an iterable *of something*.

Reply:
Okay. Yes, I went ahead and added it.